addons/account/models/account_payment_term.py

```diff
@@ -21,6 +22,22 @@ def _default_line_ids(self):
     company_id = fields.Many2one('res.company', string='Company')
     sequence = fields.Integer(required=True, default=10)
 
+    # -------Early payment discount fields-------
+    has_early_payment = fields.Boolean(string=_("Apply Early Payment Discount"), default=False)
+    percentage_to_discount = fields.Float(string=_("Discount"), default=1)
```

Comment:
I'm pretty sure we discussed about the tax machin bidule truc discount in Belgium and we agreed to let that for later

Reply:
Yes we did, however this field is the percentage to discount with or without the tax thingie, and in simpler cases (tax in/excluded), so I don't get what makes you think of that here. Did I forget a field making a reference to it during refactoring ?